recipes/crow/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conans import ConanFile, tools, CMake
+import os
+
+required_conan_version = ">=1.28.0"
+
+class CrowConan(ConanFile):
+    name = "crow"
+    homepage = "https://github.com/ipkn/crow"
+    description = "Crow is C++ microframework for web. (inspired by Python Flask)"
+    topics = ("conan", "web", "microframework", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
+    exports_sources = ["patches/*"]
+    license = "BSD3"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("openssl/1.1.1h")
+        self.requires("boost/1.69.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "crow-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+        tools.patch(**self.conan_data["patches"][self.version])
```

Comment:
Should be done in build/package

Reply:
The problem is, the original CMake fails without the patch to exclude shipped tests and examples. If I do it in the package function, I cannot build the amalgamation header anymore

Reply:
move to build() method. All content from source folder will be copied to there. Use package folder only for header-only projects which doesn't require any build step